app/src/sharedTest/java/org/oppia/app/topic/revisioncard/RevisionCardFragmentTest.kt

```diff
@@ -64,6 +71,67 @@ class RevisionCardFragmentTest {
     }
   }
 
+  @Test
```

Comment:
Add one more test case to check that when you click on `Return to topic` it actually goes to `TopicRevisionFragment`, for that you might need to start with the `TopicActivity` as the starting point instead of `RevisionCardActivity `

Reply:
I am doing it currently, if possible please provide an example

Reply:
Done, thanks 👍